qiskit_experiments/data_processing/data_processor.py

```diff
@@ -69,48 +103,79 @@ def is_trained(self) -> bool:
 
         return True
 
-    def __call__(self, data: Union[Dict, List[Dict]], **options) -> Tuple[Any, Any]:
+    def __call__(self, data: Union[Dict, List[Dict]], **options) -> Tuple[np.ndarray, np.ndarray]:
         """
         Call self on the given datum. This method sequentially calls the stored data actions
         on the datum.
 
         Args:
-            data: The data, typically from ExperimentData.data(...), that needs to be processed.
-            This dict or list of dicts also contains the metadata of each experiment.
+            data: The data, typically from ``ExperimentData.data(...)``,
+                that needs to be processed. This dict or list of dicts also contains
+                the metadata of each experiment.
             options: Run-time options given as keyword arguments that will be passed to the nodes.
 
         Returns:
             processed data: The data processed by the data processor.
```

Comment:
Can you give a bit more detail on the return type? Is it going to be an array of ufloats or is some conversion going to happen?

Reply:
no this is tuple of ndarrays as before. No return type of public method has been changed. This will be changed in follow up PR (once curve analysis supports ufloat).

Reply:
ok